website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -110,6 +110,7 @@ Each namespace rule can include a coarse-grained `policy` field, a fine-grained
 `capabilities` field, a `variables` block, or all three.
 
 The `policy` field for namespace rules can have one of the following values:
+- `deny` - prevent any capabilities
```

Comment:
```suggestion
- `deny`: prevent any capabilities
```

We seem inconsistent about this throughout the page, but this is at least consistent within this section 😅